### DISCUSSION TITLE
Is there a way to include an eraser feature in npm install?

### DISCUSSION THREAD

**Discussion by ilya2204** [fabricjs/fabric.js#7223] [Q&A]
I use fabric in my angular project, I install it via npm install. Is there some easy way to enable the eraser feature in my project? I am asking, since it is very painful to include a custom fabric.min.js build into an angular project.

**Comment by asturur** [comment]
If you are installing with NPM, you can define a post install step for your app.
In that step, you can change directory to node_modules/fabric and run the custom build command.
